src/components/Button/StatusButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { GestureResponderEvent, Pressable, StyleSheet } from 'react-native';
+
+import { selectStatus, useTheme } from '@/hooks';
+import { Status } from '@/types/status';
+
+type Props = React.ComponentPropsWithRef<typeof Pressable> & {
+  status: Status;
+  onPress: (event: GestureResponderEvent) => void;
+};
+
+const StatusButton = (param: Props) => {
+  const { themedStyles } = useTheme();
+  const actualStatus: Status = 'Inaccessible'; //added temporarily until the selected status is supported
+  const buttonStatus: Status = param.status;
+
+  return (
+    <Pressable
+      style={[
+        styles.button,
+        actualStatus === param.status
+          ? { backgroundColor: themedStyles.buttonActive }
+          : { backgroundColor: themedStyles.buttonBackground },
+      ]}
+    >
+      {selectStatus(buttonStatus)}
+    </Pressable>
+  );
+};
+export default StatusButton;
+
+const styles = StyleSheet.create({
+  button: {
+    justifyContent: 'center',
+    width: 232,
+    height: 44,
+    borderRadius: 10,
+    paddingLeft: 10,
+    marginTop: 10,
+    shadowColor: '#000',
```

Comment:
starajmy się nie używać hardcored colors tylko używajmy zmiennych z pliku `style.cs`. Co prawda nie jest on jeszcze uzupełniony zmiennymi zgodnymi z figmą, ale nie powielajmy złego podejścia

Reply:
Jak coś to wszystko jest opisane w readme